src/rpc/handlers/AccountTx.cpp

```diff
@@ -108,7 +108,8 @@ AccountTxHandler::process(AccountTxHandler::Input input, Context const& ctx) con
         {
             return Error{Status{RippledError::rpcINVALID_PARAMS, "containsLedgerSpecifierAndRange"}};
```

Comment:
I'd remove the curlies now that it's a separate one-liner `if` 👍

Reply:
Let me do it in my next PR